### PR TITLE
Added simple error throwing

### PR DESCRIPTION
**Pull request by AmasaDelano**

### CODE REVIEW

File: src/action-parser.ts [hunk 1]
```diff
@@ -117,8 +117,7 @@ function parseLocation(location: string, takingFaction: RootFaction): RootLocati
     return +location;
   }
 
-  console.error(`Could not parse location: "${location}"`);
-  return null as RootLocation;
+  throw `Could not parse location: "${location}"`;
```
Comment (seiyria) [inline src/action-parser.ts:120]: Can you do `throw new Error(msg)` instead? Throwing strings, while possible, isn't trackable.

Comment (AmasaDelano) [reply src/action-parser.ts]: Yes, done! 👍

**Comment by AmasaDelano** [comment]
This one's ready to merge.

**Comment by seiyria** [comment]
sorted in 1.0.7

**Pull request merged**